dojo/tools/govulncheck/parser.py

```diff
@@ -0,0 +1,63 @@
+import json
+from itertools import groupby
+from itertools import islice
+import logging
+from dojo.models import Finding
+
+logger = logging.getLogger(__name__)
+
+SEVERITY = 'Info'
+NO_IMPACT = "In your code no call of these vulnerable function, but they in call stack of other function"
+
+
+class GovulncheckParser:
+
+    def get_scan_types(self):
+        return ["Govulncheck Scanner"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Govulncheck Scanner findings in JSON format."
+
+    @staticmethod
+    def get_location(data, node):
+        return [f"{x['Pos']['Filename']}:{x['Pos']['Line']}:{x['Pos']['Column']}" for x in
+                data['Calls']['Functions'][str(node)]['CallSites'] if x['Parent'] == 1]
+
+    def get_findings(self, scan_file, test):
+        findings = []
+        scan_data = scan_file.read()
```

Comment:
Please use the direct form:
```Python
json.load(scan_file)`
```

Reply:
HI! @damiencarol
Thanks for your review!
I need to use read() to further remove extra text in the report before the json data. After that, the cleaned data is fed to the loads() input
Direct load() from a file object gives an error due to the fact that the report stores not only data in json format, but also text at the beginning
Is it possible to do without direct load() ?